doc/qbk/03_core/_core.qbk

```diff
@@ -96,5 +96,8 @@ effect:
 [include 5_buffers.qbk]
 [include 6_files.qbk]
 [include 7_composed.qbk]
-[8_conf_macros.qbk]
+[endsect]
+
+[section:config Configuration]
```

Comment:
We need to be careful making new sections or renaming existing sections, as the hyperlink is formed from the section name and becomes permanent. If we then change the section afterwards, any 3rd party sites (reddit, StackOverflow, et. al.) will have broken links.

Reply:
Noted. 
Is this name acceptable?